site/docs/references/custom-selectors.md

```diff
@@ -0,0 +1,181 @@
+---
+id: references/custom-selectors
+title: Custom Selectors
+layout: docs
+---
+
+You use [Custom selectors](https://drafts.csswg.org/css-extensions/#custom-selectors) to define an alias that can match complex selectors. 
+
+For example, a specific type of button within a form that appears only when hovered can be defined as a custom selector. By defining the button as a custom selector with its own name, the button is exposed in the API and available for use by using just the custom selector name.
+
+You could also use custom selectors to define a group of selectors with one name. For example, you can access all the headings on a page as one custom selector. This could be useful if you want to style just their color the same.
+
+## Usage
+
+The following code maps the alias name `controlBtn` that matches any `.btn` CSS class nested under the `.controls` CSS class.
+
+****
+```css
+/* CSS */
+@namespace "Comp";
+@custom-selector :--controlBtn .controls .btn;
+/* 
+selector: .Comp__root .Comp__controls .Comp__btn 
+*/
+:--controlBtn { border: 1px solid grey; }
+/* 
+selector: .Comp__root .Comp__controls .Comp__btn:hover 
+*/
+:--controlBtn:hover { border-color: red; }
+```
+
+### Expose pseudo-element
+
+Custom selectors generate a [pseudo-element](./pseudo-elements.md). So, for example, [importing](./imports.md) a stylesheet into another stylesheet enables access to the `form-input` pseudo-element.
```

Comment:
form-input should be controlBtn
and we should reference the prev example in the text

Reply:
Done